### PR TITLE
Renamed `Tech Reconstructor` to `Nano Reconstructor`.

### PR DESCRIPTION
**Pull request by Dzierzan**

### CODE REVIEW

**Comment by Mailaender** [comment]
All other tech buildings consist of two words.

**Comment by Dzierzan** [comment]
As far as I remember, you were removing word `tech` from all captureable `tech` buildings. How come `two words` is a rule here?

**Comment by Mailaender** [comment]
Extractor (what does it extract) → Resource Extractor is more explainable. I agree that "Tech Reconstructor" can be improved. I didn't come up with something better yet.

File: mods/hv/rules/buildings.yaml [hunk 1]
```diff
@@ -2412,7 +2412,7 @@ RECONSTRUCTOR:
 	Selectable:
 		Bounds: 2048, 2457, 0, -204
 	Tooltip:
-		Name: Tech Reconstructor
+		Name: Reconstructor
```
Comment (Mailaender) [inline mods/hv/rules/buildings.yaml:2415]: Nano Reconstructor
Universal Reconstructor

something like that

Comment (Mailaender) [reply mods/hv/rules/buildings.yaml:2415]: Reconstructor Hub/Facility/Center?

Comment (Dzierzan) [reply mods/hv/rules/buildings.yaml:2415]: Nano Reconstructor sounds OK.

**Pull request merged**